greeter/README.md

```diff
@@ -1,21 +1,706 @@
+   import {asGreeter, Greeter} from '@loopback/example-greeter-extension';
+   ```
+
+3. Add the `FrenchGreeter` class.
+
+   ```ts
+   /**
+    * A greeter implementation for French.
+    */
+   @bind(asGreeter)
+   export class FrenchGreeter implements Greeter {
+     // By checking the value of property language, the extension point matches a greeter to the language set from a request.
+     language = 'fr';
+
+     constructor(
+       /**
+        * Inject the configuration for FrenchGreeter
+        */
+       @config()
+       private options: FrenchGreeterOptions = {nameFirst: true},
+     ) {}
+
+     greet(name: string) {
+       if (this.options && this.options.nameFirst === false) {
+         return `Bonjour，${name}！`;
+       }
+       return `${name}, Bonjour`;
+     }
+   }
+   ```
+
+4. Add the `FrenchGreeterOptions` interface.
+
+   ```ts
+   /**
+    * Options for the Frech greeter
+    */
+   export interface FrenchGreeterOptions {
+     // Name first, default to `true`
+     nameFirst: boolean;
+   }
+   ```
+
+### Step 2: Bind the FrenchGreeter in the Application
+
+In `src/application.ts`, inside the constructor, add the line below:
+
+```ts
+this.component(GreetingComponent);
+// add this line
+this.add(createBindingFromClass(FrenchGreeter));
+```
+
+Add the following import statements:
+
+```ts
+import {ApplicationConfig, createBindingFromClass} from '@loopback/core'; //Change this line
+import {FrenchGreeter} from './greeter-fr'; //Add this line
+```
+
+### Step 3: Try Out the New Greeter
+
+Let's try out the new French Greeter.
+
+1. Stop the application by entering Ctrl+C.
+2. Restart the application again by running `npm start`.
+3. Go to API Explorer, http://localhost:3000/explorer.
+4. Under the `GET /greet/{name}` endpoint, enter `fr` as the language as shown below.
+
+   ![Test French Greeter](images/french-greeter-test.png)
+
+5. In the response body section, you should be able to see the greeting in French.
+   ```json
+   {
+     "timestamp": "2019-10-01T18:47:37.096Z",
+     "language": "fr",
+     "greeting": "Diana, Bonjour"
+   }
+   ```
+
+### Step 4: Configure the Greeter
+
+Besides from compositing your app with modules and extensions, LB4 also allows you to configure bound items in the context easily. Such configurations can be resolved and injected in the same way of other forms of dependencies.
+
+We will show how to configure the way it prints the Chinese greeting message with name first, e.g. `"LoopBack, hello"`. (The default setting is greeting word `"Hello"` first, e.g. `"hello, LoopBack"`)
+
+Add this line in your `application.ts` file
+
+```ts
+this.component(GreetingComponent);
+this.add(createBindingFromClass(FrenchGreeter));
+
+// Add the following line
+this.configure('greeters.ChineseGreeter').to({nameFirst: false});
+```
+
+### Step 5: Try it Out!
+
+Restart your application, try endpoint `GET /greeting/{name}`, with `Accept-Language` set to `zh`.
```

Comment:
> Restart your application, try endpoint `GET /greeting/{name}`, with `Accept-Language` set to `zh`.

Restart your application, try endpoint `GET /greeting/{name}`, with `language` set to `zh`.